core/src/autogluon/core/searcher/bayesopt/utils/test_objects.py

```diff
@@ -4,16 +4,83 @@
 Object definitions that are used for testing.
```

Comment:
where is `test_objects` being used in tests?

Reply:
./bayesopt/test_bayesopt_ei.py:from autogluon.core.searcher.bayesopt.utils.test_objects import \
./bayesopt/test_common.py:from autogluon.core.searcher.bayesopt.utils.test_objects import \
./bayesopt/test_gaussproc.py:from autogluon.core.searcher.bayesopt.utils.test_objects import default_gpmodel, \
./bayesopt/test_gp_components.py:from autogluon.core.searcher.bayesopt.utils.test_objects import \